dvc/remote/gdrive.py

```diff
@@ -393,6 +396,23 @@ def _gdrive_download_file(
         ) as pbar:
             gdrive_file.GetContentFile(to_file, callback=pbar.update_to)
 
+    @contextmanager
+    @_gdrive_retry
+    def open(self, path_info, mode="r", encoding=None):
+        assert mode in {"r", "rt", "rb"}
```

Comment:
should we raise an illegalargument here?

Reply:
It's not done in other places in the code - would have to do everywhere if needed

Reply:
@efiop what do you think?

Reply:
@shcheklein Sorry or the delay. I think we could leave it with the assert for now, same as in other places.